src/main/java/org/myrobotlab/framework/repo/Repo.java

```diff
@@ -360,19 +360,19 @@ static public void info(String format, Object... args) {
     publishStatus(Status.info(format, args));
   }
 
-  synchronized public void install() {
+  synchronized public void install() throws Exception {

```

Comment:
Can the exception type be more specific?

Reply:
updated